src/Pages/CardEditor.js

```diff
@@ -212,19 +240,20 @@ export const CardEditor = () => {
 						</Layer>
 					</Stage>
 				</div>
-				<span title="pdf" className="" onClick={() => exportToPDF()}>
-					PDF
-				</span>
+				<button title="pdf" className="button" 
+					onClick={exportToPDF}
+				>Previewer
+				</button>
 			</Wrapper>
-			<button type="button" onClick={postPdfAndCheckout}>
+			{/* <button type="button" onClick={postPdfAndCheckout}>
```

Comment:
This commented section should also go into check-out component

Reply:
Yes, it is included in PDFPreview file comment is now gone